src/schemas/21-526EZ/schema.js

```diff
@@ -1,9 +1,404 @@
+      type: 'string',
+      enum: [
+        // Are these exact?
+        'international',
+        'domestic',
+        'military'
+      ]
+    }
+  }
+};
+
+// Apparently we don't need a suffix here.
+const fullNameDef = _.omit('properties.suffix', definitions.fullName);
+
 let schema = {
   $schema: 'http://json-schema.org/draft-04/schema#',
   title: 'SUPPLEMENTAL CLAIM FOR COMPENSATION (21-526EZ)',
   type: 'object',
-  definitions: {},
-  properties: {},
+  definitions: {
+    email: definitions.email,
+    directDeposit: directDepositDef,
+    date: definitions.date,
+    // dateRange: definitions.dateRange // hopefully we can use this later
+    phone: definitions.phone,
+    fullPhone: {
+      type: 'object',
+      properties: {
+        areaNbr: {
+          type: 'string'
+          // Validation?
+        },
+        phoneNbr: {
+          $ref: '#/definitions/phone'
+        }
+      }
+    },
+    fullName: definitions.fullName
+  },
+  properties: {
+    veteran: {
+      type: 'object',
+      properties: {
+        emailAddress: {
+          $ref: '#/definitions/email'
+        },
+        alternateEmailAddress: {
+          $ref: '#/definitions/email'
+        },
+        mailingAddress: addressDef,
+        primaryPhone: {
+          $ref: '#/definitions/fullPhone'
+        },
+        forwardingAddress: _.set('properties.efctvDt', {
+          $ref: '#/definitions/date'
+        }, addressDef),
+        homelessness: {
+          type: 'object',
+          properties: {
+            hasPointOfContact: {
+              type: 'boolean'
+            },
+            pointOfContact: {
+              type: 'object',
+              properties: {
+                pointOfContactName: {
+                  type: 'string'
+                  // maxLength: 51 // Until we know otherwise?
+                },
+                // Should we make this a sub-object called
+                // pointOfContactPhone so we can re-use a definition
+                // for when the validation changes?
+                pointOfContactAreaCode: {
+                  type: 'string'
+                  // What kind of validation goes here?
+                },
+                pointOfContactPhoneNumber: {
+                  type: 'string'
+                  // What kind of validation goes here?
+                }
+              }
+            }
+          }
+        },
+        serviceNumber: {
+          type: 'string'
+          // What kind of validation goes here?
+        }
+      }
+    },
+    // Should this be an array of objects?
+    attachments: {
+      type: 'object',
+      // What kind of validation do we use for all of these?
+      docName: {
+        type: 'string'
+      },
+      dateUploaded: {
+        type: 'string'
+      },
+      attachmentType: {
+        type: 'string'
+      },
+      inflightId: {
+        type: 'string'
+      }
+    },
+    militaryPayments: {
+      type: 'object',
+      properties: {
+        payments: {
```

Comment:
The plural sort of implies there can be more than one, but it's not an array type.

Reply:
No idea. :man_shrugging: 
I'll make it an array and when we run into it, we can change it if needed.